src/openeo_grass_gis_driver/register_actinia_processes.py

```diff
@@ -85,8 +83,7 @@ def register_processes():
                 OPENEO_ACTINIA_ID_DICT[process] = {"id": actiniaid}
                 ACTINIA_OPENEO_PROCESS_DESCRIPTION_DICT[process] = module
 
-        # TODO: add logger
-        print("... successfully registered modules!")
+        log.info("... successfully registered modules!")
 
     else:
-        print('... error registering modules!')
+        log.info('... error registering modules!')
```

Comment:
```suggestion
        log.error('... error registering modules!')
```

?

Reply:
Makes sense, yes, I changed it. In this case, the code will just continue and the GRASS GIS modules will not be visible in the backend, but it is in general functional and all processes directly implemented in the openeo-ggd can be used.